README.md

```diff
@@ -316,31 +316,17 @@ _See code: [src/commands/upgrade/index.ts](src/commands/upgrade/index.ts)_
 
 # Installation
 
-There are 2 ways to get ahold of the CLI.
-
-## NPM
-
----
-
-**Prerequisites**
-* NodeJS >= 18
-* NPM
-* Linux environment (others may work, but are untested)
-
----
+## Obtaining prebuilt binaries
 
-**Steps**
-1. Clone this repo
-2. cd into the cloned repo
-3. Run the following command:
+You can obtain prebuilt binaries of the CLI which have no dependency to NodeJS. These can be found [here](https://github.com/eclipse-velocitas/cli/releases).
+Currently we are supporting only linux `arm64` and `x64` architecture. This package is already pre-intalled with our [devcontainer-base-image](https://github.com/eclipse-velocitas/devcontainer-base-images)
 
-```sh-session
-$ npm install -g velocitas-cli
+If you need to install it on custom setup, please use follow:
+```
+curl -L https://github.com/eclipse-velocitas/cli/releases/download/v0.5.5/velocitas-linux-x64 -o /usr/bin/velocitas
```

Comment:
please change the version and the architecture with a placeholder e.g <VERSION_TAG> / <ARCH>